pkg/query/columnquery.go

```diff
@@ -271,13 +272,41 @@ func (q *ColumnQueryAPI) Query(ctx context.Context, req *pb.QueryRequest) (*pb.Q
 		}
 	}()
 
+	if req.GetReportType() == pb.QueryRequest_REPORT_TYPE_PROFILE_METADATA {
+		mappingFiles, err := q.GetMappingFiles(ctx, req.GetMerge())
+
+		labels, labels_err := q.Labels(ctx, &pb.LabelsRequest{
```

Comment:
the labels we want here are not quite the same, we need to replicate what `.Labels` does but for the pprof profile label columns.